server/middleware/checkPrisonerInCaseloadMiddleware.ts

```diff
@@ -21,6 +27,29 @@ export default function checkPrisonerInCaseload({
     const globalSearchUser = userHasRoles([Role.GlobalSearch], userRoles)
     const canViewInactiveBookings = userHasRoles([Role.InactiveBookings], userRoles)
     const inactiveBooking = ['OUT', 'TRN'].some(prisonId => prisonId === prisonerData.prisonId)
+    const pomUser = userRoles.filter(role => role.includes('POM')).length > 0
```

Comment:
Can we use the `userHasRoles` method here to keep consistent with the above role checks?